app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+
+const {
+  getTopics
+} = require('./controllers/topics.js');
+
+
+app.get('/api/topics', getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status) res.status(err.status).send({ msg: err.msg });
```

Comment:
This error handler is good, however it's not relevant at this stage in the project. As the code is right now, this will never be called. Worth taking out and re-introducing later on to a relevant ticket. Try not to over-engineer too early, instead focus on the ticket itself.